### ISSUE TITLE
Handling PSQ expanded query length by 1024 (machine generated queries are long)

### ISSUE THREAD

**Issue by cramraj8** [hltcoe/patapsco#53]
@cash , I remember there was a version of Patapsco where the query length is also truncated and after expanded with PSQ terms also the new query is truncated by 1024 tokens. But when I run the current GitHub version, I get the error `: too many boolean clauses org.apache.lucene.queryparser.classic.ParseException` . Do you know if we have to manually raise a flag to handle that ?

**Comment by cash** [comment]
Yes, I need to merge that in from the other repo. Sorry for the delay on that!

**Comment by cramraj8** [comment]
Thanks, we are working on the BETTER project submission which is incorporating patapsco. If you could fast process this by today, that will be really helpful.

**Comment by cash** [comment]
@cramraj8 those commits are now pushed to master!
